### ISSUE TITLE
ui: create physical query plan inspector admin view

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#1439]
PR: https://github.com/cockroachdb/cockroach/pull/15908

**Comment by Amruta-Ranade** [comment]
As per discussion with Diana, not to be documented for 1.1

**Comment by dianasaur323** [comment]
nope, let's wait on this

**Comment by jseldess** [comment]
@dianasaur323, do you mean we should punt to 2.1?

**Comment by dianasaur323** [comment]
@jseldess yep, currently the physical query plan inspector is really built for internal use, and we didn't get the chance to clean it up in this release. So let's do it in the next release once we do some clean up work. In fact, this issue might be a bit preemptive since we aren't sure what the actual feature will look like yet

**Comment by jseldess** [comment]
Closing in favor of related work in 2.1.
